src/Klarna.Checkout/KlarnaCheckoutService.cs

```diff
@@ -364,13 +382,16 @@ public virtual AddressUpdateResponse UpdateAddress(ICart cart, AddressUpdateRequ
                 OrderTaxAmount = AmountHelper.GetAmount(totals.TaxTotal),
                 OrderLines = GetOrderLines(cart, totals, configuration.SendProductAndImageUrl),
                 PurchaseCurrency = cart.Currency.CurrencyCode,
-                ShippingOptions = configuration.ShippingOptionsInIFrame ? GetShippingOptions(cart, cart.Currency, ContentLanguage.PreferredCulture) : Enumerable.Empty<ShippingOption>()
+                ShippingOptions = configuration.ShippingOptionsInIFrame ? GetShippingOptions(cart, cart.Currency, ContentLanguage.PreferredCulture) : Enumerable.Empty<ShippingOption>(),
```

Comment:
Could you please split into separate rows? It is too long.

```suggestion
                ShippingOptions = configuration.ShippingOptionsInIFrame
                    ? GetShippingOptions(cart, cart.Currency, ContentLanguage.PreferredCulture)
                    : Enumerable.Empty<ShippingOption>(),
```